hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/OzoneManagerStateMachine.java

```diff
@@ -314,8 +313,7 @@ public TransactionContext preAppendTransaction(TransactionContext trx)
   @Override
   public CompletableFuture<Message> applyTransaction(TransactionContext trx) {
     try {
-      OMRequest request = OMRatisHelper.convertByteStringToOMRequest(
-          trx.getStateMachineLogEntry().getLogData());
+      final OMRequest request = (OMRequest) trx.getStateMachineContext();
```

Comment:
Please add a line that this case depends on the deserialization done in preAppendTransaction.

Reply:
Updated.  Thanks.